vendor/github.com/docker/libnetwork/iptables/conntrack.go

```diff
@@ -16,7 +16,7 @@ var (
 
 // IsConntrackProgrammable returns true if the handle supports the NETLINK_NETFILTER and the base modules are loaded
 func IsConntrackProgrammable(nlh *netlink.Handle) bool {
```

Comment:
putting `!darwin` at the top line (as freebsd libnetwork port does) would be better.

Reply:
This was not just enough because of this: vendor/github.com/docker/libnetwork/drivers/bridge/setup_ip_tables.go:362:2: undefined: iptables.DeleteConntrackEntries

Reply:
please try to describe (here) how FreeBSD port does.

Reply:
This issue arises from the code definition of a bridge driver that is being used by Libnetwork. What freebsd port has done is that, they are having a separate bridge driver for freebsd usage and using the ip_tables related codes from there.

Reply:
I think we also need to port libnetwork.  Is it possible?

Reply:
If we look at the current code, we can almost say that libnetwork is almost ported. The os build dependency is almost covered.  As once suggest by you, we just need a bridge mimicking something like eth0. 

I'm putting all the libnetwork related changes to one commit. Maybe later on if we decide to have a different repo, this will be useful.